prog/vm/github_runner.rb

```diff
@@ -200,6 +201,23 @@ def quota_available?
       Clog.emit("allowed because of low utilization") { {exceeded_concurrency_limit: {family_utilization:, label: github_runner.label, repository_name: github_runner.repository_name}} }
     end
     github_runner.log_duration("runner_capacity_waited", Time.now - github_runner.created_at)
+
+    hop_apply_custom_label_quota if GithubCustomLabel.where(label: github_runner.label, installation_id: github_runner.installation_id).any?
+    hop_allocate_vm
+  end
+
+  label def apply_custom_label_quota
+    label = github_runner.label
+    installation_id = github_runner.installation_id
+    custom_label = GithubCustomLabel.where(label:, installation_id:).first
+    hop_allocate_vm unless custom_label
```

Comment:
We shouldn’t encounter this case. I think we can assume that `custom_label` exists here. If it doesn’t, we should allow the runner to fail or we destroy it.